ably-iosTests/ARTRealtimeResumeTest.m

```diff
@@ -40,39 +40,41 @@ - (void)tearDown {
 
 
 /**
-  create 2 connections, each connected to the same channel.
- disonnect and reconnect one of the connections, then use that channel
- to send and recieve message. verify all messages sent and recieved ok.
+ Create 2 connections, each connected to the same channel.
+ Disconnect and reconnect one of the connections, then use that channel to send and recieve message.
+ Verify all messages sent and recieved ok.
```

Comment:
This is not what the test should be doing as far as I am aware.  What we want it to do is:

* Client A & Client B connect, attach to channel Y
* Client A is forcibly disconnected and does not (yet) reconnect and attempt resume
* Client B then publishes some messages on channel Y, and waits until the messages are received on channel Y
* Client A reconnects and resumes the connection.  As connection resume should be working, it then receives the messages on channel Y whilst the client was disconnected, and channel Y of course remains attached